xclim/core/options.py

```diff
@@ -136,14 +136,16 @@ class set_options:
     - ``missing_options``: Dictionary of options to pass to the missing method. Keys must the name of
         missing method and values must be mappings from option names to values.
 
-    You can use ``set_options`` either as a context manager:
-    >>> import xclim
-    >>> import xarray
-    >>> ds  = xarray.open_dataset("path_to_tas_file").tas
-    >>> with xclim.set_options(metadata_locales=['fr'])
+    Examples
+    --------
+    # You can use ``set_options`` either as a context manager:
+    >>> import xclim as xc
+    >>> from xclim.testing import open_dataset
+    >>> ds  = open_dataset(path_to_tas_file).tas
```

Comment:
I'm wondering why you are exposing the `xclim.testing` functions here. They should be visible using the `xr` accessor, according to the doctest namespace aliases. In other words
```suggestion
    >>> ds  = xr.open_dataset(path_to_tas_file).tas
```

Reply:
Woups, I forgot about that. This test was failing because of its formatting, but I "fixed" the `open_dataset` too. Will revert that.